server/sonar-server/src/test/java/org/sonar/server/project/ws/UpdateVisibilityActionTest.java

```diff
@@ -280,17 +281,24 @@ public void execute_throws_ISE_when_project_organization_uuid_does_not_match_exi
   public void execute_changes_private_flag_of_specified_project_and_all_children_to_specified_new_visibility() {
     ComponentDto project = randomPublicOrPrivateProject();
     boolean initiallyPrivate = project.isPrivate();
+
+    BranchDto branchDto = ComponentTesting.newBranchDto(project);
+    dbClient.branchDao().insert(dbSession, branchDto);
+
+    ComponentDto branch = ComponentTesting.newProjectBranch(project, branchDto);
```

Comment:
Do you mean, to verify that new branches inherit the same visibility from the project? That was already the case before my change, so I assumed that was tested. Or do you mean something else?